### PR TITLE
Tpetra:  temporary workaround for #5179

### PR DESCRIPTION
**Pull request by kddevin**
@trilinos/tpetra 

## Description
This is a temporary workaround for the most reproducible error of the errors in #5179 .
As we debug the issue, we will reverse this workaround.

## Motivation and Context
In response to user deadlines, we hope this workaround will enable users to make progress.

## How Has This Been Tested?
Tpetra tests on linux all pass
Using instructions for running MueLu_Driver.cpp in #5179, type 1 failures were eliminated.

### CODE REVIEW

File: packages/tpetra/core/src/Tpetra_Details_FixedHashTable_def.hpp [hunk 1]
```diff
@@ -76,7 +76,14 @@ namespace FHT {
...
 template<class ExecSpace>
 bool worthBuildingFixedHashTableInParallel () {
-  return ExecSpace::concurrency() > 1;
+//  KDD 8/19:  A temporary fix for #5179.  It appears some errors come from
+//  KDD 8/19:  computeOffsetsFromCounts; this temporary patch avoids the most
+//  KDD 8/19:  reproducible of the errors.  We'll reverse this as we debug
+//  KDD 8/19:  the problem, but this temporary fix may allow users to make 
+//  KDD 8/19:  progress toward their milestones.
+//  return ExecSpace::concurrency() > 1;
+    return false;
```
Comment (mhoemmen) [inline packages/tpetra/core/src/Tpetra_Details_FixedHashTable_def.hpp]: This makes me sad, but if it must be, it must be.

Comment (jhux2) [reply packages/tpetra/core/src/Tpetra_Details_FixedHashTable_def.hpp]: @mhoemmen This is hopefully temporary.  The change buys us time to figure out the root.

Comment (jhux2) [reply packages/tpetra/core/src/Tpetra_Details_FixedHashTable_def.hpp]: cause.

Comment (kddevin) [reply packages/tpetra/core/src/Tpetra_Details_FixedHashTable_def.hpp]: Yup, intended to be temporary so we aren't a roadblock.

Comment (mhoemmen) [reply packages/tpetra/core/src/Tpetra_Details_FixedHashTable_def.hpp]: Thanks for tracking this down!

**Pull request merged**